### DISCUSSION TITLE
Prevent row click if clicking the select cell

### DISCUSSION THREAD

**Discussion by williamyeny** [icflorescu/mantine-datatable#238] [Q&A]
When clicking the padding of a select cell, it triggers the row click event. This happens if the user misclicks the checkbox. Is there any way to prevent this?

See my crude screenshot:
![image](https://user-images.githubusercontent.com/12968964/229562333-37a4d276-50bc-490f-8016-73a2c7e29792.png)

I'd like the row click event to NOT be triggered if the red areas are clicked.

**Comment by icflorescu** [comment]
The user should be more careful :-)
Just kidding, this should be fixed in a few minutes, as soon as `v2.3.2` hits the `npm` registry.
